SHOE/Source/EditingUI.cpp

```diff
@@ -972,6 +987,228 @@ void EditingUI::GenerateEditingUI() {
 		ImGui::End();
 	}
 
+	if (*(GetMaterialWindowEnabled()))
+	{
+		ImGui::Begin("Material Editor");
+
+		int materialUIIndex = GetMaterialUIIndex();
+		std::shared_ptr<Material> currentMaterial = globalAssets.GetMaterialAtID(materialUIIndex);
+
+		std::string infoStr = currentMaterial->GetName();
+		std::string node = "Edting Material: " + infoStr;
+
+		ImGui::Text(node.c_str());
+
+		if (ImGui::ArrowButton("Previous Material", ImGuiDir_Left)) {
+			materialUIIndex--;
+			if (materialUIIndex < 0) {
+				materialUIIndex = globalAssets.GetMaterialArraySize() - 1;
+			}
+			this->materialUIIndex = materialUIIndex;
+			//globalAssets.currentSky = globalAssets.GetSkyAtID(materialUIIndex);
```

Comment:
Please remove these commented out lines - you were likely using them for reference, which is great, but now they just add clutter.

Reply:
Resolved in force pushed commit